### PR TITLE
Move develop (v1) to master

### PR DESCRIPTION
**Pull request by dahlbyk**
As discussed (https://github.com/dahlbyk/posh-git/pull/522#issuecomment-382942875), v0.x development has moved from `master` to `v0`. This bring v1 progress on `develop` into `master` as our main line of development.

We've already reviewed everything except the last two commits.

### CODE REVIEW

File: src/PoshGitTypes.ps1 [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+    [UntrackedFilesMode]$UntrackedFilesMode = [UntrackedFilesMode]::Default
+
+    [bool]$EnablePromptStatus    = !$global:GitMissing
+    [bool]$EnableFileStatus      = $true
+
+    [Nullable[bool]]$EnableFileStatusFromCache        = $null
+    [string[]]$RepositoriesInWhichToDisableFileStatus = @()
+
+    [string]$DescribeStyle = ''
+    [psobject]$WindowTitle = {param($GitStatus, [bool]$IsAdmin) "$(if ($IsAdmin) {'Administrator: '})$(if ($GitStatus) {"posh~git ~ $($GitStatus.RepoName) [$($GitStatus.Branch)] ~ "})PowerShell $($PSVersionTable.PSVersion) $([IntPtr]::Size * 8)-bit ($PID)"}
```
Comment (rkeithhill) [inline src/PoshGitTypes.ps1]: Missing changes from PR #567

**Comment by rkeithhill** [comment]
I did a dir diff on this branch with develop and there wasn't much except in some of top-level md files and the psd1 `1.0.0` vs `1.0.0.0` (branch names in URLS).  

image

**Comment by dahlbyk** [comment]
> I did a dir diff on this branch with develop and there wasn't much except in some of top-level md files and the psd1 1.0.0 vs 1.0.0.0 (branch names in URLS).

That's what I would expect. I assume most of what you see aligns with what GitHub shows on https://github.com/dahlbyk/posh-git/pull/568/commits/3e4509e74b644beae27bd10e3138c928ae4211b8?

I reverted the version back to 1.0.0.0 as an indicator that a bug report's coming from a clone of posh-git, rather than an official release (which will have 1.0.0). Or did we decide that isn't necessary?

**Comment by rkeithhill** [comment]
I "think" it's still necessary because module versioning at the PowerShell level (vs PSGallery/PowerShellGet) is still using .NET's System.Version and can't indicate pre-prelease info in the version number.

**Pull request merged**